Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ChannelPage.tsx

```diff
@@ -342,16 +360,8 @@ export default function ChannelPage(props: IProps) {
         const regex = new RegExp('\(A[0-9]+\)Analog Channel [0-9]+');
 
         const digital = regex.test(ch.Description) && ch.MeasurementType == 'Digital';
-        const sparePhrase = ch.Description != null && [
```

Comment:
we should make `SpareChannel` a requried ValueList if we do this otherwhise we won't have it and it'll be a pain to figure out where it is